libgpos/include/gpos/memory/CMemoryPoolManager.h

```diff
@@ -26,6 +26,7 @@
 #include "gpos/memory/CMemoryPool.h"
 
 
+
```

Comment:
Licensing on top of the file is outdated. I noticed you are fixing some. Then you need to fix all in the PR.